web/stats/aoc_structures.json

```diff
@@ -1169,10 +1169,10 @@
       "hp": "2750",
       "ra": "-",
       "at": "-",
-      "ar": "10/10",
-      "GA": "+1250 HP with Fortified Wall<br />(sometimes gate parallel to the bottom right edge has 0:30 bt)",
+      "ar": "6/6",
```

Comment:
I tested it with that Gate, and it seems to have 6/6 armor. The 10/10 armor gates are apparently not the ones used for damage calculation.